### PR TITLE
Add some related model caching

### PR DESCRIPTION
**Pull request by jaredlewis**
@wesleykendall

### CODE REVIEW

File: entity_event/models.py [hunk 1]
```diff
@@ -799,6 +834,8 @@ class Subscription(models.Model):
     sub_entity_kind = models.ForeignKey(EntityKind, null=True, related_name='+', default=None)
     only_following = models.BooleanField(default=True)
 
+    objects = SubscriptionManager()
```
Comment (wesleykendall) [inline entity_event/models.py]: Can you try out this new thing in Django 1.7 for this? https://docs.djangoproject.com/en/1.8/releases/1.7/#calling-custom-queryset-methods-from-the-manager

I've known about this feature for quite some time, and it looks like it is applicable for this model manager

File: entity_event/models.py [hunk 2]
```diff
@@ -715,6 +717,39 @@ def __str__(self):
...
+        """
+        Cache any related objects that we may use
+        :return:
+        """
+        return self.select_related(
+            'medium',
+            'source',
+            'entity',
+            'sub_entity_kind'
+        )
```
Comment (wesleykendall) [inline entity_event/models.py]: This easily fits on one line instead of 6, no?

return self.select_related('medium', 'source', 'entity', 'entity_kind')

**Comment by wesleykendall** [comment]
@jaredlewis done with comments

**Comment by jaredlewis** [comment]
@wesleykendall addressed those comments, im not sure how to do a release of this, not sure ive ever had to do a public release before

**Comment by wesleykendall** [comment]
@jaredlewis merge into the main master branch on the main fork after this is merged. Then do python publish.py from the main repo

**Pull request merged**